airflow/www/api/experimental/endpoints.py

```diff
@@ -41,6 +47,124 @@
 api_experimental = Blueprint('api_experimental', __name__)
 
 
+@api_experimental.route(
+    '/dags/<string:dag_id>/dag_runs/<string:execution_date>/tasks/<string:task_id>/logs',
```

Comment:
This fn should take an (optional) try_number too.

Reply:
Should the try_number be added as a query parameter?

Reply:
Sure, sounds good.